XenModel/Alerts/PolicyAlert.cs

```diff
@@ -52,7 +52,7 @@ public PolicyAlert(long priority, string name, DateTime time, string body, strin
 
             if(Type == "info")
             {
-                Text = Message.FriendlyBody(name);
+                Text = string.Format(Message.FriendlyBody(name), policyName);

```

Comment:
This one is not really needed because of all the vmss messages we have only one error message is a string format, but it does not cause any error if we leave it.